test/unit/latest_news_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class LatestNewsTest < ActiveSupport::TestCase
+  setup do
+    Setting[:latest_news_rss_feed] = File.expand_path(File.dirname(__FILE__) + '/rss_feed.xml')
```

Comment:
I don't think testing SimpleRSS parses the file & fetches items is our responsibility here, they already test that. https://github.com/cardmagic/simple-rss/blob/master/test/base/base_test.rb

Why not just expecting the `.parse` calls, which is really all that Foreman is concerned about?